docs/reference/release_notes.md

```diff
@@ -6,17 +6,37 @@ linkTitle: Release notes
 We recommend always using "tagged" versions of Ondat rather than "latest",
 and to perform upgrades only after reading the release notes.
 
-The latest tagged release is `2.5.0`. For
-installation instructions please see our
+The latest tagged release is `2.6.0`. For
+installation instructions see our
 [Install](/docs/reference/cluster-operator/install) page.
 
-The latest CLI release is `2.5.0`, available from
+The latest CLI release is `2.6.0`, available from
 [Github](https://github.com/storageos/go-cli/releases).
 
 # Upgrading
 
-To upgrade from version 1.x to 2.x, please contact [support](/docs/support) for assistance.
+To upgrade from version 1.x to 2.x, contact Ondat [support](/docs/support) for assistance.
 
+## 2.6.0 - Released 2022-02-14
+
+### New
+
+Portal Manager:
+
+- Initial release of the Portal Manager, which supports the connection to Ondat SaaS Platform.
+
+Kubectl Plugin: 
+
+- We have added a `--dry-run` flag into install command, so you can view the installation manifests written locally to `./storageos-dry-run/`.
+
```

Comment:
Add the following:

- We have added capability for airgapped installation. The new options can also be used outside of an airgapped cluster.